consensus/proto_array/src/fork_choice_test_definition.rs

```diff
@@ -131,6 +134,8 @@ impl ForkChoiceTestDefinition {
                     expected_head,
                     proposer_boost_root,
                 } => {
+                    let current_epoch =
+                        std::cmp::max(justified_checkpoint.epoch, latest_justified) + 3;
```

Comment:
Can you commit a comment with the reasoning in the PR for the fix?

Reply:
I haven't made this update yet because I'm still trying to figure out whether this hack should remain or if these tests should be re-designed. And more broadly, if we care about these scenarios, should we just upstream them to the spec repo and get rid of our maintenance burden here

Reply:
I've reverted @realbigsean's changes (see https://github.com/sigp/lighthouse/pull/5094#issuecomment-1931265465)